tests/onnxruntime/test_optimization.py

```diff
@@ -100,6 +101,23 @@ def test_optimization_details(self):
             self.assertEqual(len(sorted_operators_difference), 0)
             gc.collect()
 
+    def test_optimization_fp16(self):
+        model_name = "hf-internal-testing/tiny-random-distilbert"
+        optimization_config = OptimizationConfig(optimization_level=0, fp16=True)
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            output_dir = Path(tmp_dir)
+            model_path = output_dir.joinpath("model.onnx")
+            optimized_model_path = output_dir.joinpath("model-optimized.onnx")
+            optimizer = ORTOptimizer.from_pretrained(model_name, feature="sequence-classification")
+            optimizer.export(
+                onnx_model_path=model_path,
+                onnx_optimized_model_output_path=optimized_model_path,
+                optimization_config=optimization_config,
+            )
+            model = onnx.load(optimized_model_path.as_posix())
```

Comment:
Can you add a test about the relative difference in the output? I'm not a huge fan of changing dtype for all the layers like this.

Reply:
Not really sure why since we are not applying any custom logic. 
What if there is more difference? what do you expect from the test?

Reply:
Added the test but not sure what the benefit is